boards/shields/lmp90100_evb/doc/index.rst

```diff
@@ -0,0 +1,57 @@
+.. _lmp90100_evb_shield:
+
+LMP90100 Sensor AFE Evaluation Board
+####################################
+
+Overview
+********
+
+The Texas Instruments LMP90100 Sensor AFE Evaluation Board (EVB) is a
+development kit for the TI LMP90xxx series of analog sensor frontends.
+
+.. figure:: ./lmp90100eb_lmp90100eb.jpg
+   :width: 640px
+   :align: center
+   :alt: LMP90100 EVB
+
+   LMP90100 EVB (Credit: Texas Instruments)
+
+Requirements
+************
+
+This shield can only be used with a development board which provides a
+configuration for Arduino connectors and defines a node alias for the
+SPI interface (see :ref:`shields` for more details).
+
+The SPIO connector pins on the LMP90100 EVB can be connected to the
+Arduino headers of the development board using jumper wires.
+
+For more information about interfacing the LMP90xxx series and the
+LMP90100 EVB in particular, see these TI documents:
+
+- `LMP90100 Sensor AFE Evaluation Board User's Guide`_
+- `LMP90100 Multi-Channel, Low Power 24-Bit Sensor AFE`_
+
+Samples
+*******
+
+Zephyr RTOS includes one samples targeting the LMP90100 EVB:
```

Comment:
change ``samples`` to ``sample``

Reply:
@dbkinder Thank you for reviewing. I have fixed the two typos.